pom.xml

```diff
@@ -197,14 +228,20 @@
       <artifactId>powermock-api-mockito2</artifactId>
       <scope>test</scope>
     </dependency>
-      
+
     <!-- static analysis -->
     <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
```

Comment:
as this for fixing some upper-bounds issue?

Reply:
Yes @jtnord, the tests start to fail due to this:

```Caused by: java.io.IOException: Pipeline: Groovy v2.46 failed to load.
 - JavaScript GUI Lib: jQuery bundles (jQuery and jQuery UI) plugin v1.2 is older than required. To fix, install v1.2.1 or later.```

(if you want I can post the full stacktrace)